storage/integration_test.go

```diff
@@ -203,6 +205,22 @@ func testConfig(ctx context.Context, t *testing.T) *Client {
 	return client
 }
 
+// testConfigGPRC returns a gRPC-based client to access GCS. testConfigGRPC
+// skips the curent test when being run in Short mode.
+func testConfigGRPC(ctx context.Context, t *testing.T) (gc *Client) {
+	if testing.Short() {
+		t.Skip("Integration tests skipped in short mode")
+	}
+
+	var err error
```

Comment:
nit: don't need this on a separate line.

Reply:
Done!